readme.md

```diff
@@ -165,15 +165,15 @@ configure various timers in the global default metric registry of your applicati
   [javadoc]: https://www.javadoc.io/doc/nl.talsmasoftware.context/context-propagation-root
 
 
-  [servletrequest propagation]: servletrequest-propagation
-  [slf4j mdc propagation]: slf4j-propagation
-  [log4j2 thread context propagation]: log4j2-propagation
-  [locale context]: locale-context
-  [spring security context]: spring-security-context
-  [opentracing span propagation]: opentracing-span-propagation
-  [context propagation metrics]: context-propagation-metrics
-  [context propagation micrometer]: context-propagation-micrometer
+  [servletrequest propagation]: managers/context-manager-servletrequest
+  [slf4j mdc propagation]: managers/context-manager-slf4j
+  [log4j2 thread context propagation]: managers/context-manager-log4j2
+  [locale context]: managers/context-manager-locale
+  [spring security context]: managers/context-manager-spring-security
+  [opentracing span propagation]: managers/context-manager-opentracing
+  [context propagation metrics]: timers/context-timer-metrics
+  [context propagation micrometer]: timers/context-timer-micrometer
```

Comment:
Note: Maybe it would also be necessary to rename these references, since for example "context propagation metrics" is now called "Context timer (metrics)" (at least in the `pom.xml`).